### ISSUE TITLE
[BUG] DateField with no `step` config still processes Shift+ArrowDown and Shift+ArrowUp to try to step

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#13835]
### Description
Create a DateField using default configurations - no `step` config. Pressing Shift+ArrowDown or Shift+ArrowUp throw an error because it tries to step by the configured step increment.

### Additional Context
Possibly, even without a `step`, the keyboard actions should have a logical default action to move by one day:

```
    getStepDate(direction) {
        const step = this.step || oneDay;

        return DH.add(this.value, (direction || 1) * step.magnitude, step.unit);
    }
```

Where `oneDay` is a const in the module header:

```
const oneDay = { magnitude : 1, unit : 'day' };
```

**Comment by ExtAnimal** [comment]
> Reproduced with a failing Siesta test.
>  A fix can either supply the suggested `oneDay` default in `getStepDate` or, arguably more correct, stop contributing the trigger key bindings at all when no `step` is configured.

I would argue that it is not more correct to ignore keystrokes.

Even if there is no spare width to display the stepper triggers, it is very useful for a user to be able to bump values up and down using keyboard gestures. We could separate the step value from the presence of stepper triggers.

But regardless, the Scheduler `EventEditor` is wide enough in 7.x onwards to accommodate stepper triggers if the `format` results in strings less than 11 characters. So I suggest we include them:

<img width="1028" height="729" alt="Image" src="https://github.com/user-attachments/assets/5fdcbe2f-474e-4c1d-ba11-7eae8ae24f47" />
